test/e2e/page-objects/pages/home/non-evm-homepage.ts

```diff
@@ -32,7 +32,6 @@ class NonEvmHomepage extends HomePage {
    * Clicks the send button on the non-EVM account homepage.
    */
   async clickOnSendButton(): Promise<void> {
-    await this.driver.waitForControllersLoaded();
```

Comment:
this step doesn't relate to this method. We wait for the controllers whenever we land on the login screen (that's already done, so no need to check anymore if controllers are loaded)